projects/docs/docs/guides/environment.md

```diff
@@ -18,6 +18,13 @@ If you want to pass multiple environment variables just separate them with a spa
 TUIST_APP_NAME=MyApp TUIST_APP_LOCALE=pl tuist generate
 ```
 
+Tuist also supports loading in environment variables from a `.env` file located in the `Tuist` folder. Simply create a `.env` file in `Tuist` and prefix your environment variables using the standard Tuist convention i.e. `TUIST_XXX`:
```

Comment:
I think we should configure the location of the .env file inside the Config.swift (defaulting it to nil)

Reply:
What would be the value of that? I think location-based convention is more suitable here?

Reply:
I think it:
- would be more explicit and less magic
- being it explicit, we can fail if the file is not found instead of just ignoring it as we are doing now

Reply:
Ok, let's keep it hardcoded in `Tuist/.env` for now then 🚀